### ISSUE TITLE
[Bug]: Some commands do not respect `--profile`

### ISSUE THREAD

**Issue by peterschmidt85** [dstackai/dstack#526]
### dstack version

0.10.2

### Host OS

macOS 13.1

### Host Arch

ARM

### What happened?

It seems that some commands do not respect `--profile` and I have to use `--project` instead.

This is certainly the case with `dstack ps`, however, this also can be the case with `dstack logs`.

**Comment by r4victor** [comment]
No longer relevant after #604 – there is no need to specify the profile for commands like `dstack ps`.
